xrmocap/ops/fourd_association/builder.py

```diff
@@ -0,0 +1,14 @@
+# yapf: disable
```

Comment:
Shall we rename `xrmocap/ops/fourd_association` into `xrmocap/ops/botton_up_association`? Someone may develop a new method just like 4-D association in the future, which is in bottom-up manner, but not 4-D.

Reply:
Yes,  `fourd_association_evaluation` can be also renamed to `botton_up_association_evaluation`. Only the matching method is replaceble.